lib/liveValidation/liveValidator.ts

```diff
@@ -495,6 +488,18 @@ export class LiveValidator {
     };
   }
 
+  private transformResponseStatusCode(liveResponse: LiveResponse) {
+    // If status code is passed as a status code string (e.g. "OK") transform it to the status code
+    // number (e.g. '200').
+    if (
+      !http.STATUS_CODES[liveResponse.statusCode] &&
+      utils.statusCodeStringToStatusCode[liveResponse.statusCode.toLowerCase()]
```

Comment:
Can you add a test to validate the statusCode returned in response?

Reply:
Test added